src/identity/apis/auth.ts

```diff
@@ -283,3 +287,20 @@ export const updateDefaultOrgByAccountID = async ({
 
   // success status code is 204; no response body expected.
 }
+
+// fetch user's default account
+export const getDefaultAccount = async () => {
+  try {
+    const resp = await getAccounts({})
+    if (resp.status !== 200) {
+      throw new Error(resp.data.message)
+    }
+    const {data} = resp
+
+    if (Array.isArray(data) && data.length) {
+      return data.find(line => line.isDefault)
```

Comment:
Thanks for adding this function to the API layer!  This was going to be saved for the switch-account refactor, but I appreciate you doing it now.

Two other minor suggestions - I might pick some more descriptive var than `line` (like `acct` or `account`). And I might leave eventing out, since how we event might depend on how this function is called. At least for the new quartz APIs, we don't do that in other parts of the API layer right now - instead, we handle eventing in a thunk, or at the component level.

Reply:
yup totally meant to change the naming to something semantic. Dang it! I've updated it to `account`